tests/detect-bytemath-03/test.rules

```diff
@@ -0,0 +1,9 @@
+alert tcp any any -> any any (ipv4.hdr; byte_extract: 1, 0, extracted_val, relative;byte_math: bytes 1, offset 1, oper -, rvalue extracted_val, result var;byte_test: 1, <, var, 9;msg:\
+"Byte extract and byte math with byte test verification\
+";sid:1;)
+alert tcp any any -> any any (ipv4.hdr; byte_extract: 1, 0, extracted_val, relative;byte_math: bytes 1, offset 1, oper -, rvalue extracted_val, result var;byte_test: 1,!=, var, 9;msg:\
+"Byte extract and byte math with byte test verification\
+";sid:2;)
+alert tcp any any -> any any (ipv4.hdr; byte_extract: 1, 0, extracted_val, relative;byte_math: bytes 1, offset 0, oper -, rvalue extracted_val, result var;byte_test: 1,>, var, 0;msg:\
+"Byte extract and byte math with byte test verification\
+";sid:3;)
```

Comment:
byte_test works with unsigned values; the first rule:
- (byte_extract )Value 69 is extracted
-  (byte_match) 34 - 69 = -35
- (byte_test) 6 < -35 (where -35 is represented in a `uint64_t` value)

If the intent was to have unsigned values compared than this is ok. If the intent was to compare 6 and -35 as signed values, then the rule would require adjustment to avoid the negative value computation.

The other rules are fine:
- (2nd rule) 69 is extracted, byte_math computes -35 and then -35 is tested for inequality with 6
- (3rd rule) 69 extracted; byte_math computes 0 and then the check of 69 > 0 succeeds.

Reply:
I was not comparing unsigned values.
So no changes are needed.
Based on comments I will be renaming the directories instead of combining the tests.
Thank you for the review!